webnn/resources/test_data/cast.json

```diff
@@ -0,0 +1,3040 @@
+        "data": [
+          34,
+          83,
+          113,
+          31,
+          62,
+          80,
+          8,
+          40,
+          104,
+          42,
+          6,
+          91,
+          93,
+          21,
+          40,
+          21,
+          51,
+          110,
+          115,
+          12,
+          122,
+          68,
+          57,
+          72
+        ],
+        "type": "float16"
+      }
+    },
+    {
+      "name": "cast uint32 4D tensor to int32",
+      "inputs": {
+        "input": {
+          "shape": [2, 2, 2, 3],
+          "data": [
+            34,
+            83,
+            113,
+            31,
+            62,
+            80,
+            8,
+            40,
+            104,
+            42,
+            6,
+            91,
+            93,
+            21,
+            40,
+            21,
+            51,
+            110,
+            115,
+            12,
+            122,
+            68,
+            57,
+            72
+          ],
+          "type": "uint32"
+        }
+      },
+      "type": "int32",
+      "expected": {
+        "name": "output",
+        "shape": [2, 2, 2, 3],
+        "data": [
+          34,
+          83,
+          113,
+          31,
+          62,
+          80,
+          8,
+          40,
+          104,
+          42,
+          6,
+          91,
+          93,
+          21,
+          40,
+          21,
+          51,
+          110,
+          115,
+          12,
+          122,
+          68,
+          57,
+          72
+        ],
+        "type": "int32"
+      }
+    },
+    {
+      "name": "cast uint32 4D tensor to int64",
+      "inputs": {
+        "input": {
+          "shape": [2, 2, 2, 3],
+          "data": [
+            34,
+            83,
+            113,
+            31,
+            62,
+            80,
+            8,
+            40,
+            104,
+            42,
+            6,
+            91,
+            93,
+            21,
+            40,
+            21,
+            51,
+            110,
+            115,
+            12,
+            122,
+            68,
+            57,
+            72
+          ],
+          "type": "uint32"
+        }
+      },
+      "type": "int64",
+      "expected": {
+        "name": "output",
+        "shape": [2, 2, 2, 3],
+        "data": [
+          "34",
+          "83",
+          "113",
+          "31",
+          "62",
+          "80",
+          "8",
+          "40",
+          "104",
+          "42",
+          "6",
+          "91",
+          "93",
+          "21",
+          "40",
+          "21",
+          "51",
+          "110",
+          "115",
+          "12",
+          "122",
+          "68",
+          "57",
+          "72"
+        ],
+        "type": "int64"
+      }
+    },
+    {
+      "name": "cast uint32 4D tensor to int8",
+      "inputs": {
+        "input": {
+          "shape": [2, 2, 2, 3],
+          "data": [
+            34,
```

Comment:
At least one negative value would be prudent.

Reply:
The data of **uint32** type couldn't be a negative number. Do you suggest test negative data for other casting? Thanks.